src/test/java/games/strategy/engine/data/annotations/InvalidFieldNameExample.java

```diff
@@ -16,15 +16,15 @@ protected InvalidFieldNameExample(final String name, final Attachable attachable
   }
 
   // should have been prefixed with "m_". Should cause test to fail.
```

Comment:
This comment should probably be updated to reflect that it's the fact that the field has a different name which is causing the test to fail rather than the lack of the `m_` prefix.